tests/e2e/ctl_v3_move_leader_test.go

```diff
@@ -88,13 +101,12 @@ func testCtlV3MoveLeader(t *testing.T, cfg e2e.EtcdProcessClusterConfig) {
 		}
 	}
 
-	os.Setenv("ETCDCTL_API", "3")
-	defer os.Unsetenv("ETCDCTL_API")
```

Comment:
this was spouting a warning, so I thought I'd clean this up. Let me know if I should leave it.

Reply:
`ETCDCTL_API` is 3 by default, why the case need this? It should be legacy "issue". So it's Ok to remove it.